apps/andi/assets/css/datasets.scss

```diff
@@ -62,3 +61,13 @@
 .datasets-table__cell--break {
   word-break: break-word;
 }
+
+.datasets-table__th--asc{
+    -webkit-box-shadow: inset 0 0.2em 0 0 $fadedblack;
+    box-shadow: inset 0 0.2em 0 0 $fadedblack;
+}
+
+.datasets-table__th--desc {
+    -webkit-box-shadow: inset 0 -3px 0 0 $fadedblack;
+    box-shadow: inset 0 -3px 0 0 $fadedblack;
```

Comment:
this one is px but the one above is em?  Should they both be rems or ems?

Reply:
reeeeeeeems